### ISSUE TITLE
Rename svFSIplus to svMultiPhyisics 

### ISSUE THREAD

**Issue by ktbolt** [SimVascular/svFSIplus#305]
### Use Case

It is time to rename svFSIplus to svMultiPhyisics.

### Problem

none

### Solution

I will first make changes to the CMake files. 

### Alternatives considered

none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct and Contributing Guidelines

**Comment by ktbolt** [comment]
I've renamed svFSIplus in the CMake files. Building now creates
```
CMakeCache.txt		Externals-build/	Makefile		svMultiPhysics-build/
CMakeFiles/		Externals-prefix/	cmake_install.cmake	svMultiPhysics-prefix/
```
and `svMultiPhysics-build/bin/svmultiphysics`

**Comment by ktbolt** [comment]
I've replaced the `svfsi`  Code/ThirdParty directories names 
```
README		metis_svfsi/	parmetis_svfsi/	tetgen/
eigen/		msmpi/		share/		tinyxml/
```
with `internal`
```
README.md		gklib_internal/		msmpi/			share/			tinyxml/
eigen/			metis_internal/		parmetis_internal/	tetgen/
```

It took me a `very very very long time to understand` how to do this:  variable names related to these directory names are created automatically in some CMake macros using a specific format. The directory names are also used in different forms, upper case, lower case, and different positions in a variable
```
THIRDPARTY_METIS_INTERNAL
metis_internal
METIS_INTERNAL_LIBRARY_NAME
SV_LIB_THIRDPARTY_METIS_INTERNAL_NAME
SV_USE_INTERNAL_METIS
USE_INTERNAL_METIS
```

In addition to these variables and the directory names themselves are used all over the place, ended up needing to change five files, a bunch of directory names and a bunch variable names. 

I will document this in the CMake code.

**Comment by mrp089** [comment]
Happy the renaming is happening! I had a brief look at your branch, @ktbolt. The paths in `tests/conftest.py` still contain the `svfsiplus` executable.

The tests can't run anyways because they fail to activate the `svmultiphysics` conda environment because the Docker images still have the `svfsiplus` environment. However, the Docker containers will only be updated once the branch is merged into main (which it can't because the tests are failing). Seems like a 🐥 vs. 🥚 sort of problem. 

@dcodoni, is this correct, and is there a good way to resolve this?

**Comment by dcodoni** [comment]
I will have to modify the dockerfile that builds the solver. Also I would need to modify the workflow files to be consistent with the new path to executable. The conda environment can still be named svfsiplus it should not create any problems. If you want to rename that too, I will have to rebuild the container with the libraries.
My suggestion would be to perform all this operation without testing, just merge everything needed and when everything is consistent with the new nomenclature then we can run the tests.

**Comment by ktbolt** [comment]
Thanks @mrp089 for noticing the missed name changes, I've updated the files and the README too.

**Comment by ktbolt** [comment]
Merged into main.
